Packs/SecurityIntelligenceServicesFeed/Integrations/SecurityIntelligenceServicesFeed/SecurityIntelligenceServicesFeed.yml

```diff
@@ -0,0 +1,139 @@
+category: Data Enrichment & Threat Intelligence
+commonfields:
+  id: Security Intelligence Services Feed
+  version: -1
+configuration:
+- display: S3 Access Key
+  name: accessKey
+  type: 4
+  required: true
+- display: S3 Secret Key
+  name: secretKey
+  type: 4
+  required: true
+- display: Feed Type
+  name: feedType
+  type: 16
+  options:
+  - Host
+  - Domain
+  - Scam
+  - Phishing
+  - Malware
+  - Content
+  defaultvalue: Domain
+  required: true
+- display: Fetch indicators
+  name: feed
+  defaultvalue: true
+  type: 8
+  required: false
+- display: Indicator Reputation
+  name: feedReputation
+  defaultvalue: feedInstanceReputationNotSet
+  type: 18
+  required: false
+  options:
+  - None
+  - Good
+  - Suspicious
+  - Bad
+  additionalinfo: Indicators from this integration instance will be marked with this
+    reputation
+- display: Source Reliability
+  name: feedReliability
+  defaultvalue: F - Reliability cannot be judged
+  type: 15
+  required: true
+  options:
+  - A - Completely reliable
+  - B - Usually reliable
+  - C - Fairly reliable
+  - D - Not usually reliable
+  - E - Unreliable
+  - F - Reliability cannot be judged
+  additionalinfo: Reliability of the source providing the intelligence data
+- display: ""
+  name: feedExpirationPolicy
+  defaultvalue: indicatorType
+  type: 17
+  required: false
+  options:
+  - never
+  - interval
+  - indicatorType
+  - suddenDeath
+- display: ""
+  name: feedExpirationInterval
+  defaultvalue: "20160"
+  type: 1
+  required: false
+- display: Feed Fetch Interval
+  name: feedFetchInterval
+  defaultvalue: "60"
+  type: 19
+  required: false
+- display: First Fetch Time Range (<number> <time unit>, e.g., 12 hours, 7 days, 3 months, 1 year)
+  additionalinfo: All indicators from the given day(s) will be fetched.
+  name: firstFetchInterval
+  defaultvalue: Today
+  type: 0
+  required: true
+- display: Bypass exclusion list
+  name: feedBypassExclusionList
+  defaultvalue: ""
+  type: 8
+  required: false
+  additionalinfo: When selected, the exclusion list is ignored for indicators from
+    this feed. This means that if an indicator from this feed is on the exclusion
+    list, the indicator might still be added to the system.
+- display: Trust any certificate (not secure)
+  name: insecure
+  defaultvalue: "false"
+  type: 8
+  required: false
+- display: Use system proxy settings
+  name: proxy
+  defaultvalue: "false"
+  type: 8
+description: A PassiveTotal with Security Intelligence Services Feed provides you
```

Comment:
please add feedTags param to the yml and add implementation.
- additionalinfo: Supports CSV values.
  display: Tags
  name: feedTags
  required: false
  type: 0
you can see an example in this pr:
https://github.com/demisto/content/pull/8284